src/main/java/com/radixdlt/crypto/encryption/PrivateKeyEncrypter.java

```diff
@@ -80,7 +81,7 @@ public static String createEncryptedPrivateKey(String password) throws GeneralSe
         return gson.toJson(keystore);
     }
 
-    public static byte[] decryptPrivateKey(String password, Reader keyReader) throws IOException, GeneralSecurityException {
+    public static HashCode decryptPrivateKey(String password, Reader keyReader) throws IOException, GeneralSecurityException {
```

Comment:
I don't think we should be using HashCode class as holder of private key information. This can quickly be abused and exposes a lot of methods on the private key which we don't want to support.

Reply:
@talekhinezh I've used Hasher in engine-library (Atom and Universe classes). Perhaps better to move it there rather than core?